application/testing/CMakeLists.txt

```diff
@@ -500,9 +500,11 @@ f3d_test(NAME TestInteractionAnimationFrameRate DATA f3d.glb ARGS --animation-pr
 f3d_test(NAME TestInteractionAnimationCameraMovement DATA CameraAnimated.glb ARGS --camera-index=0 --animation-progress INTERACTION)#Space;MouseMovement;Space;
 f3d_test(NAME TestInteractionAnimationInvert DATA f3d.glb ARGS --animation-speed-factor=-1 --animation-progress INTERACTION)#Space;Wait;Space;
 
-# Test scrolling cheatsheet with no model by mouse wheel and by using scrollbar
-f3d_test(NAME TestInteractionNoModelScrollWheel RESOLUTION 1000,600 NO_DATA_FORCE_RENDER INTERACTION UI)
-f3d_test(NAME TestInteractionNoModelScrollBar RESOLUTION 1000,600 NO_DATA_FORCE_RENDER INTERACTION UI)
+if(VTK_VERSION VERSION_GREATER_EQUAL 9.3.20240729)
```

Comment:
What was the stuff added in VTK? Link the PR maybe?

Reply:
SSIM comparison. for some reason the baseline doesnt work with the legacy comparison algorithm ?